src/OVAL/fts_sun.h

```diff
@@ -30,6 +30,7 @@
  */
 
 #include "config.h"
+#include "oscap_platforms.h"
```

Comment:
why? it's gonna be included as part of config.h

Reply:
@mpreisler haha... you are right. Should I use `config.h` for the others instead of `oscap_platforms.h`?

Reply:
@mpreisler ^

Reply:
every c file should include config as its first include and no header file should include it, that's the general rule

Reply:
@mpreisler fixed.